db/seeds.rb

```diff
@@ -11,8 +14,8 @@ def from_csv(file_path)
 
 def seed_city_database(file_path)
   csv = from_csv(file_path)
-  cities = csv.each do |row|
-    row.keep_if {|k,v| k == :city}
+  cities = csv.each do |city|
```

Comment:
We can definitely change it. I had it like that so it was easier for me to read through all the methods.